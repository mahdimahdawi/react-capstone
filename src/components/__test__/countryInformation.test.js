import renderer from "react-test-renderer";
import CountryInformation from "../countryInformation";
import { Provider } from "react-redux";
import store from "../../redux/cinfigureStore";

test("Render <CountryInfromation /> Component", () => {
  const comp = renderer
    .create(
      <Provider store={store}>
        <CountryInformation />
      </Provider>
    )
    .toJSON();
  expect(comp).toMatchSnapshot();
});
