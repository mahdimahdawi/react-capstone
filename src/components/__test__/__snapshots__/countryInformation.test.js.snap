// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <CountryInfromation /> Component 1`] = `
<div
  className="fetch-data"
>
  <h1>
    Fetching data...
  </h1>
  <div
    class="loader"
  >
    <div
      class="inner one"
    />
    <div
      class="inner two"
    />
    <div
      class="inner three"
    />
  </div>
</div>
`;
